packages/@aws-cdk/aws-lambda/test/test.lambda.ts

```diff
@@ -1178,6 +1178,25 @@ export = {
     test.done();
   },
 
+  'environment variables are prohibited in China'(test: Test) {
```

Comment:
Add a test for what happens when the region is undefined and when the region is a token